src/components/DataDetails.vue

```diff
@@ -20,7 +20,8 @@ v-card.data-details
     template(#row="props")
       v-row(justify="space-between")
         v-col.shrink.py-1.body-2.font-weight-bold {{ props.datum.key }}
-        v-col.shrink.py-1.body-2 {{ props.datum.value }}
+        v-col.py-1
+          v-row.mx-0.body-2(justify="end") {{ props.datum.value }}
```

Comment:
I definitely could have missed something here, but would simply remove the `.shirnk` works as well?

Reply:
Sort of, but that isn't exacly what I wanted.  In the "after" example above, `a` would be right next to `bvar` instead of right-aligned.  I wanted to have justify-content: space-between while still giving multi-line values room.

Reply:
re: adding text-right

That also isn't quite what I wanted.  The word "lazy" would end up right-aligned.

Reply:
Hm, I see the problem of adding `.text-right` but still wondering if the wrapping element is necessary.

Reply:
Yeah I tried to avoid it too...  But I believe the wrapping element is necessary to produce exactly this layout.  You need the inner text box to grow to its natural width and then right-align the text container rather than the text itself.

There may be some CSS grid magic to avoid the extra node, but grid magic would be less readable than this.

Reply:
Hm, I found a combination that works. `.py-1.body-2.d-flex.justify-end`

Reply:
> .d-flex

Brilliant!  Fixed.

Reply:
Cool, learned something more about text-align with multiple lines. 😉